[2021-08-10 02:32:09,887] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-10T02:31:59.977766+00:00 [queued]>
[2021-08-10 02:32:09,906] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-10T02:31:59.977766+00:00 [queued]>
[2021-08-10 02:32:09,907] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:32:09,907] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-10 02:32:09,907] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:32:09,915] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-08-10T02:31:59.977766+00:00
[2021-08-10 02:32:09,917] {standard_task_runner.py:52} INFO - Started process 222952 to run task
[2021-08-10 02:32:09,928] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-08-10T02:31:59.977766+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp9w6p0twi', '--error-file', '/tmp/tmppcy4qbf7']
[2021-08-10 02:32:09,940] {standard_task_runner.py:77} INFO - Job 32: Subtask task_4
[2021-08-10 02:32:10,104] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-08-10T02:31:59.977766+00:00 [running]> on host airflowvm
[2021-08-10 02:32:10,254] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T02:31:59.977766+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T02:31:59.977766+00:00
[2021-08-10 02:32:10,255] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-10 02:32:10,256] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-08-10 02:32:10,279] {subprocess.py:75} INFO - Output:
[2021-08-10 02:32:13,282] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-10 02:32:13,316] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210810T023159, start_date=20210810T023209, end_date=20210810T023213
[2021-08-10 02:32:13,346] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-10 02:32:13,381] {local_task_job.py:151} INFO - Task exited with return code 0
